docs/learn/advanced/18-syncing.md

```
@@ -0,0 +1,63 @@
+    trust_height: 0
+    trust_hash: ""
+    trust_period: 168h0m0s
+    discovery_time: 15s
+    temp_dir: ""
+    chunk_request_timeout: 10s
+    chunk_fetchers: "4"
+```
+
+### Checking if sync is complete
+
+- Query for the node status using the REST or GRPC API
```

Markdownlint (0.37.0)

Expected: asterisk; Actual: dash
Unordered list style

(MD004, ul-style)